diff --git a/codex-rs/tui/src/collab.rs b/codex-rs/tui/src/collab.rs
index b6c7d8095..02632a74b 100644
--- a/codex-rs/tui/src/collab.rs
+++ b/codex-rs/tui/src/collab.rs
@@ -24,6 +24,8 @@ pub(crate) fn spawn_end(ev: CollabAgentSpawnEndEvent) -> PlainHistoryCell {
         new_thread_id,
         prompt,
         status,
+        agent_type: _,
+        agent_name: _,
     } = ev;
     let new_agent = new_thread_id
         .map(|id| Span::from(id.to_string()))
