diff --git a/codex-rs/cli/src/main.rs b/codex-rs/cli/src/main.rs
index defc063eb..022d6bbdf 100644
--- a/codex-rs/cli/src/main.rs
+++ b/codex-rs/cli/src/main.rs
@@ -110,11 +110,9 @@ enum Subcommand {
     Completion(CompletionCommand),
 
     /// Run commands within a Codex-provided sandbox.
+    #[clap(visible_alias = "debug")]
     Sandbox(SandboxArgs),
 
-    /// Debugging tools.
-    Debug(DebugCommand),
-
     /// Execpolicy tooling.
     #[clap(hide = true)]
     Execpolicy(ExecpolicyCommand),
@@ -152,36 +150,6 @@ struct CompletionCommand {
     shell: Shell,
 }
 
-#[derive(Debug, Parser)]
-struct DebugCommand {
-    #[command(subcommand)]
-    subcommand: DebugSubcommand,
-}
-
-#[derive(Debug, clap::Subcommand)]
-enum DebugSubcommand {
-    /// Tooling: helps debug the app server.
-    AppServer(DebugAppServerCommand),
-}
-
-#[derive(Debug, Parser)]
-struct DebugAppServerCommand {
-    #[command(subcommand)]
-    subcommand: DebugAppServerSubcommand,
-}
-
-#[derive(Debug, clap::Subcommand)]
-enum DebugAppServerSubcommand {
-    // Send message to app server V2.
-    SendMessageV2(DebugAppServerSendMessageV2Command),
-}
-
-#[derive(Debug, Parser)]
-struct DebugAppServerSendMessageV2Command {
-    #[arg(value_name = "USER_MESSAGE", required = true)]
-    user_message: String,
-}
-
 #[derive(Debug, Parser)]
 struct ResumeCommand {
     /// Conversation/session id (UUID) or thread name. UUIDs take precedence if it parses.
@@ -457,15 +425,6 @@ fn run_execpolicycheck(cmd: ExecPolicyCheckCommand) -> anyhow::Result<()> {
     cmd.run()
 }
 
-fn run_debug_app_server_command(cmd: DebugAppServerCommand) -> anyhow::Result<()> {
-    match cmd.subcommand {
-        DebugAppServerSubcommand::SendMessageV2(cmd) => {
-            let codex_bin = std::env::current_exe()?;
-            codex_app_server_test_client::send_message_v2(&codex_bin, &[], cmd.user_message, &None)
-        }
-    }
-}
-
 #[derive(Debug, Default, Parser, Clone)]
 struct FeatureToggles {
     /// Enable a feature (repeatable). Equivalent to `-c features.<name>=true`.
@@ -734,11 +693,6 @@ async fn cli_main(codex_linux_sandbox_exe: Option<PathBuf>) -> anyhow::Result<()
                 .await?;
             }
         },
-        Some(Subcommand::Debug(DebugCommand { subcommand })) => match subcommand {
-            DebugSubcommand::AppServer(cmd) => {
-                run_debug_app_server_command(cmd)?;
-            }
-        },
         Some(Subcommand::Execpolicy(ExecpolicyCommand { sub })) => match sub {
             ExecpolicySubcommand::Check(cmd) => run_execpolicycheck(cmd)?,
         },
