diff --git a/codex-rs/app-server-protocol/schema/typescript/v2/TurnStartParams.ts b/codex-rs/app-server-protocol/schema/typescript/v2/TurnStartParams.ts
index 8cf1aaf20..4987c9e5c 100644
--- a/codex-rs/app-server-protocol/schema/typescript/v2/TurnStartParams.ts
+++ b/codex-rs/app-server-protocol/schema/typescript/v2/TurnStartParams.ts
@@ -14,37 +14,37 @@ export type TurnStartParams = { threadId: string, input: Array<UserInput>,
 /**
  * Override the working directory for this turn and subsequent turns.
  */
-cwd?: string | null, 
+cwd: string | null, 
 /**
  * Override the approval policy for this turn and subsequent turns.
  */
-approvalPolicy?: AskForApproval | null, 
+approvalPolicy: AskForApproval | null, 
 /**
  * Override the sandbox policy for this turn and subsequent turns.
  */
-sandboxPolicy?: SandboxPolicy | null, 
+sandboxPolicy: SandboxPolicy | null, 
 /**
  * Override the model for this turn and subsequent turns.
  */
-model?: string | null, 
+model: string | null, 
 /**
  * Override the reasoning effort for this turn and subsequent turns.
  */
-effort?: ReasoningEffort | null, 
+effort: ReasoningEffort | null, 
 /**
  * Override the reasoning summary for this turn and subsequent turns.
  */
-summary?: ReasoningSummary | null, 
+summary: ReasoningSummary | null, 
 /**
  * Override the personality for this turn and subsequent turns.
  */
-personality?: Personality | null, 
+personality: Personality | null, 
 /**
  * Optional JSON Schema used to constrain the final assistant message for this turn.
  */
-outputSchema?: JsonValue | null, 
+outputSchema: JsonValue | null, 
 /**
  * EXPERIMENTAL - set a pre-set collaboration mode.
  * Takes precedence over model, reasoning_effort, and developer instructions if set.
  */
-collaborationMode?: CollaborationMode | null, };
+collaborationMode: CollaborationMode | null, };
